src/Sylius/Behat/Context/Api/Shop/OrderContext.php

```diff
@@ -34,21 +35,31 @@ final class OrderContext implements Context
     /** @var ApiClientInterface */
     private $productsClient;
 
+    /** @var ApiClientInterface */
+    private $productsShopClient;
```

Comment:
IMO the default one in shop context should be service for shop client, so there should be `$productsAdminClient` and `$productsClient` (or `$productsShopClient` to make it more clear)